src/client/components/EditUserForm/EditUserForm.tsx

```diff
@@ -107,7 +107,14 @@ const EditUserForm: React.FC<Props> = ({ user, canEditPermissions, canEditRoles,
       {[RoleName.NATIONAL_CORRESPONDENT, RoleName.ALTERNATE_NATIONAL_CORRESPONDENT, RoleName.COLLABORATOR].includes(
         userRole?.role
       ) &&
-        roleToEdit && <UserRolePropsFields role={roleToEdit} onChange={changeUserRoleProp} enabled={enabled} />}
+        roleToEdit && (
+          <UserRolePropsFields
+            role={roleToEdit}
+            onChange={changeUserRoleProp}
+            defaultCountryIso={countryIso}
```

Comment:
What is meant by defaultCountryIso ? Can you avoid to pass it as prop and get it directly where needed ?

Reply:
done